zrml/prediction-markets/src/benchmarks.rs

```diff
@@ -414,7 +389,7 @@ benchmarks! {
     internal_resolve_scalar_reported {
         let total_accounts = 10u32;
         let asset_accounts = 10u32;
-        let (_, market_id) = setup_resolve_common_scalar_after_dispute::<T>(total_accounts, asset_accounts)?;
+        let (_, market_id) = setup_resolve_common_scalar_after_dispute::<T>(asset_accounts)?;
     }: {
         let market = T::MarketCommons::market(&market_id)?;
         let disputes = crate::Disputes::<T>::get(&market_id);
```

Comment:
I would expect a benchmark of `PredictionMarket::on_resolution` instead of `T::SimpleDisputes` here. In addition to that I would expect the status `Reported` (because of the benchmark name `internal_resolve_scalar_reported`), but a dispute already happened with `setup_resolve_common_scalar_after_dispute`